packages/subgraph/src/permissions.mapping.ts

```diff
@@ -139,18 +139,20 @@ export function handleScopeFunction(event: ScopeFunction): void {
 
   // create new parameter or override old one
   for (let i = 0; i < event.params.paramType.length; i++) {
-    const paramType = PARAMETER_TYPE[event.params.paramType[i]]
-    const paramComp = PARAMETER_COMPARISON[event.params.paramComp[i]]
-    const compValue = event.params.compValue[i]
-
-    const parameterId = getParameterId(functionId, i)
-    const parameter = new Parameter(parameterId)
-    parameter.owningFunction = functionId
-    parameter.index = i
-    parameter.type = paramType
-    parameter.comparison = paramComp
-    parameter.comparisonValue = [compValue]
-    parameter.save()
+    if (event.params.isParamScoped[i]) {
```

Comment:
looking at this code I'm wondering if the subgraph correctly handles the case of unscoping a previously scoped parameter... 🤔

Reply:
does the app actually rely on this change to be released? or is it more of a cleanup chore?

Reply:
> looking at this code I'm wondering if the subgraph correctly handles the case of unscoping a previously scoped parameter... 🤔

Yes, here is the unscoping: https://github.com/gnosis/zodiac-modifier-roles/blob/93f6aec8e2e4ba17e79c7b7b1a72fb4df5ca7b45/packages/subgraph/src/permissions.mapping.ts#L251-L267

Reply:
> does the app actually rely on this change to be released? or is it more of a cleanup chore?

It is required, or the frontend will think that the parameters are scoped.

Reply:
> Yes, here is the unscoping:

Right, that handles the case of unscoping a param via the more fine grained `unscopeParameter` function. However, it's also possible to set scopings on all parameters at once using the `scopeFunction` function. It seems the subgraph would only handle updates to the param scopings if these params continue to be scoped. If we have a param scoping with `isScoped: false` we must remove that parameter entry from the subgraph.